public/tools/clientSetup4RMT.sh

```diff
@@ -0,0 +1,306 @@
+#! /bin/sh
+
+WGET=/usr/bin/wget
+OPENSSL=/usr/bin/openssl
+CP=/bin/cp
+CAT=/bin/cat
+CHMOD=/bin/chmod
+CUT=/usr/bin/cut
+GREP=/usr/bin/grep
+RM=/bin/rm
+SUSECONNECT=/usr/bin/SUSEConnect
+GPG=/usr/bin/gpg
+SUPPORTCONFIG=/etc/supportconfig.conf
+SUPPORTCONFIGENTRY=VAR_OPTION_UPLOAD_TARGET
+SED=/usr/bin/sed
+
+CA_TRUSTSTORE="/etc/ssl/certs/"
+CA_GEN_TRUSTSTORE_CMD="/usr/bin/c_rehash"
+
+if [ -d "/etc/pki/trust/anchors/" ] && [ -x "/usr/sbin/update-ca-certificates" ]; then
+    CA_TRUSTSTORE="/etc/pki/trust/anchors/"
+    CA_GEN_TRUSTSTORE_CMD="/usr/sbin/update-ca-certificates"
+elif [ -d $CA_TRUSTSTORE ] && [ -x $CA_GEN_TRUSTSTORE_CMD ]; then
+    CA_GEN_TRUSTSTORE_CMD="$CA_GEN_TRUSTSTORE_CMD $CA_TRUSTSTORE"
+fi
+
+function usage()
+{
+    if [ -n "$1" ] ; then
```

Comment:
What is this? Is this required?

Reply:
Should print the first argument to STDERR if one is present, but yeah -- it seems like it isn't needed, considering that no arguments are allowed.